manuscript/encoding.md

```diff
@@ -149,7 +149,7 @@ You may have noticed the double quotes enclosing the `Hello World!` string in th
 [String Literal](https://en.wikipedia.org/wiki/String_literal "String Literal")
 : The representation of a string value within the source code of a computer program.
 
-String literals in C\* such as `"Hello World!"` make it convenient to read and write source code that needs to output text, for example. We make extensive use of string literals in `selfie.c` with [strings for reporting compiler errors](http://github.com/cksystemsteaching/selfie/blob/6069120cb8d50fd31880f69e7f0f83c387913140/selfie.c#L357-L384) as just one example.
+String literals in C\* such as `"Hello World!"` make it convenient to read and write source code that needs to output text, for example. We make extensive use of string literals in `selfie.c` with [strings for reporting compiler errors](http://github.com/cksystemsteaching/selfie/blob/6069120cb8d50fd31880f69e7f0f83c387913140/selfie.c#L2003) as just one example.
```

Comment:
@cponcelets I am not sure why you think this is a bug? The link works for me.

Reply:
Yes but It pointed to something different than string literals, or maybe you had a different aim